### DISCUSSION TITLE
🚀 Release 2.0.0 is available

### DISCUSSION THREAD

**Discussion by dominikamann** [dominikamann/oekofen-pellematic-compact#9] [Announcements]
### **Features:**

- Enables configuration of the number of heating circuits 
- Enables configuration of solar/sun support

![grafik](https://user-images.githubusercontent.com/29973737/217624206-831311b6-7284-49c9-bc29-8928bd65de6f.png)

### **Warnig:**

New configuration options have been added which may cause a break. Please check if your system still works after the update. If not, please delete your device and configure it again.

<hr /><em>This discussion was created from the release <a href='https://github.com/dominikamann/oekofen-pellematic-compact/releases/tag/v2.0.0'>🚀 Enables configuration of the number of heating circuits and solar/sun support</a>.</em>

**Comment by dominikamann** [comment]
@lucaferrucci this affects your installation. You have to set "Num of heating circuits (HKx)" to 2 and activate "Solar & Sun yes/no (SE1, SK1)". I suggest to delete your current heater and then add it again with the correct settings. You will not lose any data. 
![grafik](https://user-images.githubusercontent.com/29973737/217629497-96323a5b-05df-4396-ad3f-fb8388eec2ae.png)

**Comment by loicguiraud** [comment]
it's works perfectly, many thanks

**Reply by dominikamann** [reply]
Let me know if you need anything on top 😉
